app/furniture/marketplace/checkouts/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= turbo_frame_tag(checkout.marketplace) do %>
```

Comment:
Just read about how `turbo_frame_tag` creates a unique id from an object. Very rails-y!

Reply:
Yep! It uses `dom_id` under the covers; which I leverage quite a bit to encourage model / view consistency. Theoretically it means that drilling down from `inspect element` should be a lot easier...